CONTRIBUTING.md

```diff
@@ -2,33 +2,22 @@
 
 Contributions are highly welcome, however, except for very small changes, kindly file an issue and let's have a discussion before you open a pull request.
 
-## Building The Project
+## Building the Project
 
 Clone this repo:
 
-```bash
-git clone https://github.com/tonerdo/coverlet
-```
+    git clone https://github.com/tonerdo/coverlet.git
+    cd coverlet
 
-Change directory to repo root:
+Building, testing, and packing use all the standard dotnet commands:
 
-```bash
-cd coverlet
-```
+    dotnet build
+    dotnet test /p:CollectCoverage=true /p:CoverletOutputFormat=opencover /p:Include=[coverlet.*]*"
+    dotnet pack
 
-Execute build script:
+To see code coverage results while testing, run with a few extra parameters:
```

Comment:
We can remove this section...command test above is the same now

Reply:
Done